test/failures_testsuite/testcases.py

```diff
@@ -202,3 +201,18 @@ def test008_stop_tlog_upload_download(self):
         md5_after = self.download_file(file_name=self.file_name, keep_trying=True)
         self.assertEqual(md5_after, md5_before)
 
+    def test009_zdb_down(self):
+        """
+        - down zdb process  and make sure it will restart automatically.
```

Comment:
replace 'down' --> 'kill' .. same for test10

Reply:
done